### PR TITLE
HACKY motherduck integration for runtime olap driver

### PR DESCRIPTION
**Pull request by k-anshul**

### CODE REVIEW

File: runtime/drivers/motherduck/duckdb.go [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+		"INSTALL 'json'",
+		"LOAD 'json'",
+		"INSTALL 'icu'",
+		"LOAD 'icu'",
+		"INSTALL 'parquet'",
+		"LOAD 'parquet'",
+		"INSTALL 'httpfs'",
+		"LOAD 'httpfs'",
+		"SET max_expression_depth TO 250",
+	}
```
Comment (begelundmuller) [inline runtime/drivers/motherduck/duckdb.go:41]: Load motherduck extension and `SET` token from `cfg`

File: runtime/drivers/motherduck/duckdb.go [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+	if cfg.DBFilePath != "" {
+		err = os.Remove(cfg.DBFilePath)
+		if err != nil {
+			return err
+		}
+		// Hacky approach to remove the wal file
+		_ = os.Remove(cfg.DBFilePath + ".wal")
+	}
```
Comment (begelundmuller) [inline runtime/drivers/motherduck/duckdb.go:121]: Will duckdb still create `.db` files locally when using motherduck? Or could it just always pass `:memory:`?

File: runtime/drivers/motherduck/olap.go [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+	"github.com/rilldata/rill/runtime/drivers"
+	"github.com/rilldata/rill/runtime/pkg/observability"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+// Create instruments
+var (
+	meter                 = otel.Meter("github.com/rilldata/rill/runtime/drivers/duckdb")
```
Comment (begelundmuller) [inline runtime/drivers/motherduck/olap.go:21]: Update to motherduck

**Review by begelundmuller — CHANGES_REQUESTED** [review]

File: runtime/drivers/motherduck/duckdb.go (hunk 2 above)
Comment (k-anshul) [reply runtime/drivers/motherduck/duckdb.go]: No it doesn't. Also realised that we need to drop the instance specific database. Added a hacky impl to do that.

**Review by begelundmuller — APPROVED** [review]

**Pull request merged**